src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+@use "../variables/variables" as *;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  padding: $padding-size;
+  width: $day-size * 7 + $padding-size;
```

Comment:
1. if you want to calculate paddings into width you should add `box-sizing: border-box`, you can add it for all elements using `*` 
2. you have 2 paddings, and you have 1px gap between every day
so you need to calculate `($day-size * 7) + ($padding-size * 2) + ($gap * 6)`

Reply:
As stated in the original pr comment, the task readme specifically states that the tests are not relevant. Because of that, I ignored the tests failing. I can fix up the project to pass the tests, but could you update the readme to show we need to take the test into consideration? Or at least confirm that they need to be passed? I have followed all the requests stated by the task, so I guess I'll skip this task till I get a response.